head/core/can_task.cpp

```diff
@@ -7,13 +7,15 @@
 #include "can/core/message_handlers/motion.hpp"
 #include "can/core/message_handlers/motor.hpp"
 #include "can/core/message_handlers/move_group.hpp"
+#include "can/core/message_handlers/presence_sensing.hpp"
 #include "can/core/messages.hpp"
 #include "common/core/freertos_message_queue.hpp"
 #include "common/core/freertos_task.hpp"
 #include "head/core/tasks.hpp"
 
 static auto& right_queues = head_tasks::get_right_queues();
 static auto& left_queues = head_tasks::get_left_queues();
+static auto& head_queues = head_tasks::get_queue_client();
```

Comment:
[nit] kind of weird naming - `head_queues` feels to me like it _includes_ both left and right. maybe `common_queues` or `control_queues` or something?